ashraesizer.py

```diff
@@ -125,16 +125,53 @@ def primaryCurveAshrae(self, peakFlowTable):
         diffPeakFlowTable = np.insert(diffPeakFlowTable, len(diffPeakFlowTable)-1, diffPeakFlowTable[len(diffPeakFlowTable)-1], axis=0)
         
     
-        peakVolume = peakFlowTable[:,1] * self.People;
+        peakVolume = peakFlowTable[:,1] * self.nPeople
        
-        primaryVol = peakVolume / self.percentUseable * (self.supplyT - self.incomingT) / \
-                (self.storageT - self.incomingT);
+        primaryVol = peakVolume / self.percentUseable * (self.supplyT_F - self.incomingT_F) / \
+                (self.storageT_F - self.incomingT_F)
         
-        accurateRecoveryTons = self.People * (diffPeakFlowTable[:len(diffPeakFlowTable),1]) / \
-                (diffPeakFlowTable[:len(diffPeakFlowTable),0]) * 60 * self.rhoCp * (self.storageT - self.incomingT)/12000;
+        accurateRecoveryTons = self.nPeople * (diffPeakFlowTable[:len(diffPeakFlowTable),1]) / \
+                (diffPeakFlowTable[:len(diffPeakFlowTable),0]) * 60 * rhoCp * (self.storageT_F - self.incomingT_F)/12000
                 
         # simpleRecoveryTons = self.People * (peakFlowTable[:len(peakFlowTable),1]) / \
-        #         (peakFlowTable[:len(peakFlowTable),0]) * 60 * self.rhoCp * (self.storageT - self.incomingT)/12000;
+        #         (peakFlowTable[:len(peakFlowTable),0]) * 60 * rhoCp * (self.storageT_F - self.incomingT_F)/12000;
                 
-        return [primaryVol, accurateRecoveryTons];
-     
\ No newline at end of file
+        return [primaryVol, accurateRecoveryTons]
+     
+    def tonsRecoveryForMaxDaily(self, peakFlowTable, accurateRecoveryTons):
```

Comment:
accurateRecoveryTons isn't used in this function, can we remove it from the function definition?

Reply:
I agree. I'll remove it.